duckduckgo.safariextension/html/global.html

```diff
@@ -219,7 +248,15 @@
             }
         }, false);
     }
-
+    
+    safari.application.addEventListener('navigate', function(evt) {
```

Comment:
I added a check. `SafariNavigateEvent` is supported in Safari 5.1+ according to their docs.
https://developer.apple.com/reference/safariextensions/safarinavigateevent